src/unit_tests/wazuh_db/test_wdb.c

```diff
@@ -752,6 +763,143 @@ void test_wdb_get_config(){
     cJSON_Delete(ret);
 }
 
```

Comment:
This is legacy, but we should add a comment like "/* wdb_exec_stmt_single_column */" as a header for the block of tests for the same method.